spec/forms/teacher_interface/delete_file_form_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe TeacherInterface::DeleteFileForm, type: :model do
+  describe "#valid?" do
+    subject(:valid?) { form.valid? }
+
+    let(:form) { described_class.new(confirm:) }
```

Comment:
```suggestion
    subject(:form) { described_class.new(confirm:) }
```

Similar comment to the other form.